.gitlab/llnl-ci-conda.yml

```diff
@@ -0,0 +1,117 @@
+stages:
```

Comment:
let's for every config have a set of variables (before `stages`)
e.g., for conda
```
variables:
  CONDA_WORK_DIR: $SDK_WORK_DIR/conda
  RADICAL_PILOT_DBURL: ${MONGODB_CONNECTION_STRING}?tlsAllowInvalidCertificates=true
  TEST_TYPE: conda
```
some variable for repeated names and some for better control (?)

Reply:
Done